src/components/NavigationBar.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import url from 'url';
+import {Container, Nav, Navbar} from "react-bootstrap";
+import {Link, NavLink} from "react-router-dom";
+import "./styles/NavigationBar.css";
+
+const logo = url.resolve(process.env.PUBLIC_URL, "/icons/se-logo-trimmed.png");
+
+const NavigationBar = () => (
+  <Navbar bg="transparent" expand="sm" collapseOnSelect >
+    <Container>
```

Comment:
Left align the logo to body text for all screen sizes with: `className="mx-3 mx-sm-auto"`

Reply:
mx-3 does this at small screen sizes 
![image](https://user-images.githubusercontent.com/41414030/82528921-b05cff00-9aee-11ea-96b3-244c3429597b.png)

Reply:
Actually, i fixed this